andela/shadowcat-backend
# #167729782-docker-setup

## What does this PR do?
- Setup dockerfiles and dockercompose scripts

## Description of Task to be completed?
- Create dockerfile for building images
- Create dockercompose scripts to enable running docker containers easy
- Delete .env.sample file because of duplication
- Write scripts in package.json to make running commands easy

## How should this be manually tested?
pull or fetch this branch
Run the following commands after you cd into the directory
``` docker-compose -f docker-compose.dev.yml up --build ``` 
To run it inside dockers(ensure you have docker and docker-compose installed
Any background context you want to provide?
No

## What are the relevant pivotal tracker stories?
[#167729782](https://www.pivotaltracker.com/story/show/167729782)

* Screenshots (if appropriate)

Questions 
No
